build_library/grub_install.sh

```diff
@@ -140,6 +140,10 @@ info "Generating ${GRUB_DIR}/load.cfg"
 ESP_FSID=$(sudo grub-probe -t fs_uuid -d "${LOOP_DEV}p1")
 sudo_clobber "${ESP_DIR}/${GRUB_DIR}/load.cfg" <<EOF
 search.fs_uuid ${ESP_FSID} root \$root
```

Comment:
Also the search for ESP can be moved from load.cfg to grub.cfg, being in here is left over from pre-secure-boot when grub.cfg was read from the ESP.